code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -426,15 +446,11 @@ GLOBAL_LIST_EMPTY(TabletMessengers) // a list of all active messengers, similar
  * The program calling this proc.
  * The message that the program wishes to display.
  */
-
 /obj/item/modular_computer/proc/alert_call(datum/computer_file/program/caller, alerttext, sound = 'sound/machines/twobeep_high.ogg')
 	if(!caller || !caller.alert_able || caller.alert_silenced || !alerttext) //Yeah, we're checking alert_able. No, you don't get to make alerts that the user can't silence.
-		return
+		return FALSE
 	playsound(src, sound, 50, TRUE)
-	visible_message(span_notice("The [src] displays a [caller.filedesc] notification: [alerttext]"))
-	var/mob/living/holder = loc
-	if(istype(holder))
-		to_chat(holder, "[icon2html(src)] [span_notice("The [src] displays a [caller.filedesc] notification: [alerttext]")]")
+	visible_message(span_notice("[icon2html(src)] [span_notice("The [src] displays a [caller.filedesc] notification: [alerttext]")]"))
```

Comment:
```suggestion
	visible_message(span_notice("[icon2html(src)] [span_notice("[src] displays a [caller.filedesc] notification: [alerttext]")]"))
```

I think this is a bug?

Reply:
![image](https://user-images.githubusercontent.com/53777086/191871276-475195d2-248b-42e7-bd1b-54bac6e52d20.png)

Tablets aren't improper or anything at the moment, so this is the right way

Reply:
oh yeah